lnbits/core/views/api.py

```diff
@@ -7,6 +7,9 @@
 from urllib.parse import ParseResult, parse_qs, urlencode, urlparse, urlunparse
 
 import httpx
+import pyqrcode
```

Comment:
Any reason you've chosen pyqrcode instead of https://pypi.org/project/qrcode/?

pyqrcode seems to be pretty much unmaintained. Last release was in mid-2016.

Reply:
Because of bloat pillow